guide/how-it-works/ecash/fedimint.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: guide
+title: Fedimint
+description: Learn the basics of Fedimint.
+nav_order: 3
+parent: Ecash
+grand_parent: How it works
+permalink: /guide/how-it-works/ecash/fedimint/
+redirect_from:
+ - /guide/ecash/fedimint
+image: https://bitcoin.design/assets/images/guide/how-it-works/ecash/fedimint.jpg
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Explains generally how fedimint works
+
+Illustration sources
+
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/fedimint.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/fedimint@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/fedimint-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/fedimint-mobile@2x.jpg"
+   alt-text = "ecash overview illustration"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# Fedimint
+Fedimint is the first bitcoin-backed ecash protocol. Fedimint decentralizes trust across a federation of guardians, ensuring that no single guardian has complete control over user funds. Fedimint allows for various Federation configurations, each catering to different needs. These configurations include **Federated Mints**, **Federated Single Guardian Mints**, and **Solo Mints (Unfederated Mints)**, each with its strengths and trade-offs.
```

Comment:
Yes, Solomints are something I recently learned that Fedimint supports. They were created as an alternative to Cashu mints. In the first review call with the Fedimint developers, they referred to it as 'using a hammer to kill a fly,' since Fedimint is primarily designed for federated mints. However, in theory, you could use it for a solo mint if you wanted to, as the protocol does support it.